### DISCUSSION TITLE
Can we have more than one projectId in the same cypress.json?

### DISCUSSION THREAD

**Discussion by MatiDiez** [cypress-io/cypress#15375] [Cypress Cloud]
So our use case is. We have two separate applications under test within the same cypress project. 
We are using cucumber on top of cypress and we have tags to run our tests separate like this: 
npx cypress-tags run -e TAGS=“@app1”
npx cypress-tags run -e TAGS=“@app2”

We are recording our suite as a whole but we now need to split the results in the dashboard. We already have a bunch of custom commands and shared steps so Ideally we would like to avoid working with two separate repos. Can we use 2 projects for each app. To run on our CI something like: 

npx cypress-tags run -e TAGS=“@app1” --record --key asdasd
npx cypress-tags run -e TAGS=“@app2” --record --key asdasd

Is that possible?

**Comment by bahmutov** [comment]
It is not possible, but you could have several Cypress config files and use different config file for each project

```
npx cypress-tags run -e TAGS=“@App1” --record --key firstKey --config-file cypress-app1.json
npx cypress-tags run -e TAGS=“@App2” --record --key secondKey  --config-file cypress-app2.json
```

**Tip:** passing the record key via environment variable CYPRESS_RECORD_KEY is better way to keep it secret
